src/ingress_definition.py

```diff
@@ -440,14 +478,25 @@ def service_port(self) -> int:
     @property
     def path_routes(self) -> List[str]:
         """Return the path routes to use for the k8s ingress."""
+        value = self._get_config("path-routes")
+        if value is not None:
```

Comment:
```suggestion
        if value:
```
I think we can't use `is` here because the default value of the `path-routes` configuration is `""`.